rules/docker-fpm-frr.mk

```diff
@@ -0,0 +1,15 @@
+# docker image for fpm-frr
+
+DOCKER_FPM_FRR = docker-fpm-frr.gz
+$(DOCKER_FPM_FRR)_PATH = $(DOCKERS_PATH)/docker-fpm-frr
+$(DOCKER_FPM_FRR)_DEPENDS += $(FRR) $(SWSS)
+$(DOCKER_FPM_FRR)_LOAD_DOCKERS += $(DOCKER_CONFIG_ENGINE)
+SONIC_DOCKER_IMAGES += $(DOCKER_FPM_FRR)
```

Comment:
in the rules/docker-fpm-gobgp.mk, you have removed "SONIC_DOCKER_IMAGES += $(DOCKER_FPM_GOBGP)", I am wondering why you still keep this in docker-fpm-frr.mk